packages/common-all/src/types/typesv2.ts

```diff
@@ -456,6 +460,10 @@ export type DCommonMethods = {
   ): Promise<UpdateNoteResp>;
   updateSchema: (schema: SchemaModuleProps) => Promise<void>;
 
+  /**
+   * Write note to metadata store and/or filesystem. This will update existing note or create new if one doesn't exist.
```

Comment:
super nit: comments run a tad long - there's this neat lightweight extension in vs code that can reflow your comments: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap

Reply:
awesome thanks, will take a look